# This hold basic information about the characterists of each sub-map the ni-slam
Header header 
float32 resolution             # The map resolution [m/cell]
uint32 width                   # Map width [cells]
uint32 height                  # Map height [cells]
geometry_msgs/Pose origin      # The origin of the map.